packages/next-on-pages/package.json

```diff
@@ -0,0 +1,75 @@
+{
+	"name": "@cloudflare/next-on-pages",
+	"version": "1.0.2",
+	"bin": "./bin/index.js",
+	"scripts": {
+		"prettier": "prettier --ignore-unknown src templates tests",
+		"prettier:check": "npm run prettier -- --check",
+		"prettier:fix": "npm run prettier -- --write",
+		"lint": "eslint src",
+		"types-check": "tsc --noEmit",
+		"build": "esbuild --bundle --platform=node ./src/index.ts --external:esbuild --external:chokidar --outfile=./dist/index.js",
+		"build:watch": "npm run build -- --watch=forever",
+		"prepare": "npm run build",
+		"publish": "npm run build && changeset publish",
+		"test:unit": "npx vitest --config vitest.config.ts"
+	},
+	"files": [
+		"bin",
+		"dist",
+		"templates"
+	],
+	"keywords": [
+		"cloudflare",
+		"cloudflare pages",
+		"edge",
+		"nextjs"
+	],
+	"license": "MIT",
+	"homepage": "https://github.com/cloudflare/next-on-pages#readme",
+	"bugs": {
+		"url": "https://github.com/cloudflare/next-on-pages/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/cloudflare/next-on-pages.git"
+	},
+	"dependencies": {
+		"acorn": "^8.8.0",
+		"ast-types": "^0.14.2",
+		"chalk": "^5.2.0",
+		"chokidar": "^3.5.3",
+		"cookie": "^0.5.0",
+		"esbuild": "^0.15.3",
+		"js-yaml": "^4.1.0",
+		"pcre-to-regexp": "^1.1.0",
+		"semver": "^7.5.1",
+		"zod": "^3.21.4",
+		"zodcli": "^0.0.4"
+	},
+	"peerDependencies": {
+		"vercel": "^30.0.0",
+		"wrangler": "^3.0.0"
+	},
+	"devDependencies": {
+		"@changesets/cli": "^2.26.0",
+		"@cloudflare/workers-types": "^4.20230404.0",
+		"@tsconfig/strictest": "^2.0.0",
+		"@types/cookie": "^0.5.1",
+		"@types/js-yaml": "^4.0.5",
+		"@types/mock-fs": "^4.13.1",
+		"@types/node": "^20.1.4",
+		"@typescript-eslint/eslint-plugin": "^5.58.0",
+		"@typescript-eslint/parser": "^5.58.0",
+		"dedent-tabs": "^0.10.3",
+		"eslint": "^8.35.0",
+		"mock-fs": "^5.2.0",
+		"p-limit": "^4.0.0",
+		"prettier": "^2.8.4",
+		"typescript": "^5.0.4",
+		"vitest": "^0.31.0",
+		"vitest-environment-miniflare": "^2.13.0",
+		"next-on-pages-tsconfig": "*",
+		"eslint-config-next-on-pages": "*"
```

Comment:
In the workers-sdk repo, it was decided that these synthetic packages should use `@cloudflare/` prefix to avoid a future problem where someone actually publishes to that package name and messes us up

Reply:
sure I'll update them 🙂👍 

(PS: but the eslint-plugin one needs to stay as is for eslint convention 😓 (so that you can just refer to it as `next-on-pages` in your eslintrc without needing to specify the whole package name) )

Reply:
See https://github.com/cloudflare/workers-sdk/blob/9fd12df8d65d50ba82b927eea48de679d622b9eb/packages/wrangler/.eslintrc.js#L3

Reply:
Yes, but I think that that's just an internal package that is then extended: https://github.com/search?q=repo%3Acloudflare%2Fworkers-sdk%20%2Feslint-config-worker%2F&type=code

What I meant is that if the package is called `eslint-plugin-next-on-pages` then in your eslintrc you can refer to its rules as `next-on-pages/rule_name` instead of having to specify the whole package name  (the `eslint-plugin` is inferred) so if i where to rename the package to `@cloudflare/eslint-plugin-next-on-pages` (or anything else that doesn't start with `eslint-plugin`) then users would have to specify rules as: `@cloudflare/eslint-pluging-next-on-pages/rule_name`

which seems much worse to me, isn't it?

<details>

<summary>qwik example</summary>

For example qwik also does the same.

They have [`@builder.io/qwik`](https://www.npmjs.com/package/@builder.io/qwik), [`@builder.io/qwik-city`](https://www.npmjs.com/package/@builder.io/qwik-city) and then without the `@builder.io` prefix [`eslint-plugin-qwik`](https://www.npmjs.com/package/eslint-plugin-qwik)

</details>

Reply:
Oh sorry. Yes. The external facing package should be named appropriately.
But any internal packages should be `@cloudflare` prefixed.
Sorry for the confusion.

Reply:
no problem, sorry maybe I caused the confusion 😅 

anyways I've updated the internal packages names 🙂👍